_episodes/04-recipe.md

```diff
@@ -205,7 +205,7 @@ Let's dissect what's happening here.
 > >    something like `/home/<username>/.esmvaltool/config-user.yml` or `~/esmvaltool_tutorial/config-user.yml`.
 > > 1. ESMValTool found the recipe in its installation directory, something like
 > >    `/home/users/username/miniconda3/envs/esmvaltool/bin/esmvaltool/recipes/examples/`
->> or if you are using a pre-installed module on a server, something like `/apps/jasmin/community/esmvaltool/ESMValTool_2.5.0/esmvaltool/recipes/examples/recipe_python.yml`
+>> or if you are using a pre-installed module on a server, something like `/apps/jasmin/community/esmvaltool/ESMValTool_2.6.0/esmvaltool/recipes/examples/recipe_python.yml`
```

Comment:
On JASMIN, I see this : 
```
>esmvaltool version
ESMValCore: 2.6.0
ESMValTool: 2.6.1.dev2+gfa73e53df
```

Reply:
Thanks for taking a look @rswamina. 
The versionning for the Tool (and also the Core) has been changed in https://github.com/ESMValGroup/ESMValTool/pull/2663 (previously discussed in https://github.com/ESMValGroup/ESMValTool/discussions/2299). For a _development_ installation, the format of the version number is now like: `{major}.{minor}.{patch}.dev{distance}+g{commit}+d{date}` which makes it easier to track which development version is used for different work. I guess you are seeing this number for the Tool on Jasmin because @valeriupredoi installed the Tool in developer mode. For a conda installation (no development installation), one still obtains:
```bash
 $ esmvaltool version
ESMValCore: 2.6.0
ESMValTool: 2.6.0
```
So it is true that `esmvaltool version` may not give the same number depending on how the Tool was installed. I would prefer to keep it simple in the tutorial and only use `2.6.0` everywhere. But I understand this might be inconsistent with output of commands run on Jasmin and that machine is used as an example throughout the episodes.

Reply:
@remi-kazeroni - I understand and I think it's ok to leave it like this. I only wanted to make a note.